src/content/blog/tailscale-router-ddev-addon.md

```diff
@@ -0,0 +1,73 @@
+---
+title: "A Simple and Secure Way to Share Your DDEV Projects with Tailscale"
```

Comment:
Does it make sense to rephrase it this way, placing the main subject at the start?:

```suggestion
title: "Tailscale for DDEV: Simple and Secure Project Sharing"
```